src/components/AlertRuleForm.tsx

```diff
@@ -141,7 +142,11 @@ type Props = {
   onSubmit: (alertValues: AlertFormValues) => Promise<FetchResponse<unknown> | undefined>;
 };
 
-export const AlertRuleForm = ({ rule, onSubmit }: Props) => {
+const CommonAlertForm = (
```

Comment:
I think there might be a couple ways of doing this that are a bit more React-y. 

CommonAlertForm should be a component (it pretty much already is, it just needs to take in a single object as it's arguments). Something like this
```
const CommonAlertForm = ({ rule, onSubmit, subForm }: Props)
```

The `subForm` prob can be the sub section of form inputs you want to render. That way you could just render `CommonAlertForm` like this:
```
<CommonAlertForm rule={alertRule} onSubmit={handleOnSubmit} subForm={ProbeDurationAlertForm} />
```

Alternatively you could leverage React.Children:
```
const CommonAlertForm = ({ rule, onSubmit, children }: Props)
```
And that would lead to 
```
<CommonAlertForm rule={alertRule} onSubmit={handleOnSubmit}>
    <ProbeDurationAlertForm rule={alertRule} ... />
</CommonAlertForm>
```

Reply:
Any alternatives/opinions @jewbetcha?

Reply:
Agree. I think the `children` method sounds good for this, and can wrap all the other forms.

Reply:
Ok, I ... made changes 😄